app/views/comparisons/heat_saver_march2024/_total.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= render 'table_title', table_number: table_number, table_title: t('comparisons.tables.total_usage') %>
+
+<%= component 'comparison_table',
+              report: report, advice_page: advice_page, table_name: table_name, index_params: index_params,
+              headers: headers, colgroups: colgroups, advice_page_tab: advice_page_tab do |c| %>
+  <% @results.each do |result| %>
+    <% c.with_row do |r| %>
+      <% r.with_school school: result.school %>
+      <% r.with_reference key: 'tariff_changed_last_year',
+                          if: result.any_tariff_changed? %>
+      <% r.with_reference key: 'electricity_change_rows',
+                          period_type_string: @period_type_string,
+                          if: result.pupils_changed %>
+      <% r.with_reference key: 'gas_change_rows',
+                          period_type_string: @period_type_string,
+                          schools_to_sentence: 'some schools',
```

Comment:
Added to make the Footnote work, but don't think this is required, so will update the footnote text and the code separately.